docs/wcs/index.rst

```diff
@@ -211,48 +242,11 @@ distortions are supported:
     for backward compatibility, ``SIP`` is handled by astropy itself
     and methods exist to handle it specially.
 
-Subsetting and Pixel Scales
-===========================
-
-WCS objects can be broken apart into their constituent axes using the
-`~astropy.wcs.WCS.sub` function.  There is also a `~astropy.wcs.WCS.celestial`
-convenience function that will return a WCS object with only the celestial axes
-included.
-
-The pixel scales of a celestial image or the pixel dimensions of a non-celestial
-image can be extracted with the utility functions
-`~astropy.wcs.utils.proj_plane_pixel_scales` and
-`~astropy.wcs.utils.non_celestial_pixel_scales`. Likewise, celestial pixel
-area can be extracted with the utility function
-`~astropy.wcs.utils.proj_plane_pixel_area`.
+WCS tools
+---------
```

Comment:
Link `WCS tools` this to a separate rst file. Add the `Subsetting and Pixel Scales` section above to that file.
Add also `Matplotlib plots with correct WCS projection` as a subsection to that file.